code/game/objects/items/storage/large_holster.dm

```diff
@@ -96,7 +96,7 @@
 	icon_state = "macheteB_holster"
 	item_state = "machete_holster"
 	flags_equip_slot = SLOT_STORE
-	storage_flags = STORAGE_FLAGS_POUCH
+	storage_flags = STORAGE_FLAGS_POUCH|STORAGE_ALLOW_QUICKDRAW
```

Comment:
give it the same flags as a usual machete holster or something because I can't get it to quickdraw on shift-click or ctrl-click when I have a gunbelt neither can I get it to quick-equip back into the holster